sched/sched/sched_processtimer.c

```diff
@@ -182,14 +182,6 @@ void nxsched_process_timer(void)
 
   clock_timer();
 
-#ifdef CONFIG_SCHED_CPULOAD_SYSCLK
-  /* Perform CPU load measurements (before any timer-initiated context
-   * switches can occur)
-   */
-
-  nxsched_process_cpuload();
```

Comment:
should we remove nxsched_process_cpuload function

Reply:
After we use wdt, we don't need to process it in each timer interrupt

Reply:
When using an external clock source, you may need to use nxsched_process_cpuload
![image](https://github.com/user-attachments/assets/24474524-372f-428f-b78d-2101e5e8b2c7)